src/super_gradients/training/transforms/transforms.py

```diff
@@ -829,7 +829,31 @@ def __call__(self, sample):
         if len(targets) == 0:
             targets = np.zeros((0, 5), dtype=np.float32)
         boxes = targets[:, :4]
-        image, boxes = _mirror(image, boxes, self.prob)
+        image, boxes = _horizontal_mirror(image, boxes, self.prob)
```

Comment:
I realize that this augmentation does not respect the crowd targets.
Can you please transform `crowd_target` as well? See DetectionRescale for an example.
For this you may want to remove random sampling from _horizontal_mirror implementation and do the if random() < self.prob inside the __call__ method.

Reply:
Hi @BloodAxe , thanks for feedback.
I've made two changes to `_mirror()`:
(1) The name -> `_horizontal_mirror`
(2) The expression, but not the functionality (I believe!) of the transform of the X coordinates:
`flipped_boxes[:, 0::2] = width - boxes[:, 2::-2]`
-> `flipped_boxes[:, [0,2]] = width - boxes[:, [2,0]]`

I made change (2) because I think directly, explicitly stating the columns is more readable than implicitly with the reverse periodic slice.

Is your comment "this augmentation does not respect the crowd targets" related to change (2)? Is it because crowd_target is expressed as a list of coordinates like [ [xyxyxyxyxy...] [xyxyxyxy...] ?

Reply:
> flipped_boxes[:, [0,2]] = width - boxes[:, [2,0]]
Looks good!

Reply:
> Is your comment "this augmentation does not respect the crowd targets" related to change (2)? Is it because crowd_target is expressed as a list of coordinates like [ [xyxyxyxyxy...], [xyxyxyxy...] ?

Essentially what you want to do is the following:
```
image, targets, crowd_targets = sample["image"], sample["target"], sample.get("crowd_targets")
if self.prob < randon.random():
    image = fliplr_image(image)
    targets = fliplr_boxes(targets)
    if crowd_targets  is not None:
      crowd_targets = fliplr_boxes(targets)
      
    ...
```

Here you may notice I introduced new functions: `fliplr_image`, `fliplr_boxes` and moved probability check to the class method itself.

Reply:
ok, so this is not related to the changes I made? It's adding additional unrelated functionality? That's fine, just trying to understand